tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
I think that contracts about meaningful error messages during failure scenarios are completely appropriate.

Of course, there isn't much point in having a bunch of tests involving users doing entirely the wrong thing, but covering the key, likely failure scenarios ensures that your users are properly informed when something goes wrong.